gmmp/settings.py

```diff
@@ -158,10 +158,15 @@
     os.path.join(BASE_DIR, "static"),
 ]
 
+
+SITE_URL = env.get('SITE_URL', 'http://localhost:8000')
+
+
 # Temporary login screen for downloading the exported data
 LOGIN_URL = "/admin/login/"
 
 # Email
+FROM_EMAIL = env.get("FROM_EMAIL", "support@gmmp.ngo")
```

Comment:
```suggestion
EMAIL_FROM = env.get("GMMP_EMAIL_FROM", "webmaster@localhost")
```

To be consistent with other email variables/site_url defaulting to localhost